src/lib/media-query/server-match-media.ts

```diff
@@ -0,0 +1,136 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {DOCUMENT} from '@angular/common';
+import {Inject, Injectable, NgZone} from '@angular/core';
+import {BehaviorSubject} from 'rxjs/BehaviorSubject';
+import {MediaChange} from './media-change';
+import {BreakPoint} from './breakpoints/break-point';
+import {Observable} from 'rxjs/Observable';
+import {MatchMedia} from './match-media';
+
+/**
+ * Special server-only class to simulate a MediaQueryList and
+ * - supports manual activation to simulate mediaQuery matching
+ * - manages listeners
+ */
+export class ServerMediaQueryList implements MediaQueryList {
+  private _isActive = false;
+  private _listeners: Array<MediaQueryListListener> = [];
+
+  get matches(): boolean {
+    return this._isActive;
+  }
+
+  get media(): string {
+    return this._mediaQuery;
+  }
+
+  constructor(private _mediaQuery: string) {}
+
+  /**
+   * Destroy the current list by deactivating the
+   * listeners and clearing the internal list
+   */
+  destroy() {
+    this.deactivate();
+    this._listeners = [];
+  }
+
+  /**
+   * Notify all listeners that 'matches === TRUE'
+   */
```

Comment:
You can collapse short comment to a single line:
```ts
/** Notify all listeners that 'matches === TRUE' */
```

Reply:
Fixed